iconoir.com/components/Footer.tsx

```diff
@@ -13,29 +32,81 @@ export function Footer() {
         </LogoIcon>
         <Logo src={'/iconoir-logo.svg'} alt={'Iconoir Logo'} />
       </LogoContainer>
-      <FooterNavigationItem
-        as={'a'}
-        href={LICENSE_LINK}
-        target={'_blank'}
-        rel={'noreferrer'}
-      >
-        License
-      </FooterNavigationItem>
+      <FooterCategories>
+        <FooterCategory category={'Project'} links={[
+          {name: 'Our Mission',url:'/support'},
```

Comment:
```suggestion
          { name: 'Our Mission', url:'/support' },
```

Would reformat all of these to look like this one ^